curator-documentation/OCP-Metering-Options.md

```diff
@@ -0,0 +1,99 @@
+# OCP Metering Options
+
+
+#### Purpose
+
+The purpose of this document is to explore OCP Metering projects/options to gain potential insights that help with identifying Curator project requirements. Additionally this document highlights the unique use cases that Curator provides in comparison with other OCP Metering Options
```

Comment:
Please include the fact that the metering tool we are building is only for a single cluster.